python/tskit/trees.py

```diff
@@ -3038,6 +3038,84 @@ def simplify(
         else:
             return new_ts
 
+    def slice(
+            self, start=None, stop=None, rescale_coordinates=True, simplify=True,
+            record_provenance=True):
+        """
+        Truncate this tree sequence, returning a tree sequence that covers a reduced
+        genomic span. Sites not covered by the sliced region are thrown away.
+
+        :param float start: The leftmost genomic position, giving the (open) start point
+            of the kept region. Information along the genome prior to this point will
+            be discarded. If None, use the start position of the current tree sequence.
```

Comment:
instead of `(open)` how about `giving the start point of the kept region (including sites at this point)`. And for `stop`, how about `giving the end point of the kept region (but not including sites at this point)`.

Reply:
Yes, "open" and "closed" are a bit technical. But it's not just sites. This is applicable even without any mutations. So if I have a TS with no sites from 0.0 - 1.0, and I slice it down to 0.0 to 0.5, then there will be no tree at `sliced_ts.at(0.5)`

Reply:
True. Can you suggest a better phrasing? "open" and "closed" are not familiar to too many people

Reply:
"Inclusive" works OK, but the antonym "exclusive" is a little confusing, I think.

Reply:
I've phrased around it instead: 
> "start: The leftmost genomic position, giving the start point
>  of the kept region. Tree sequence information along the genome prior to (but
>  not including) this point will be discarded."

and

> stop: The rightmost genomic position, giving the end point of the
> kept region. Tree sequence information at this point and further along the
> genomic sequence will be discarded.

Reply:
oh, perfect.